src/pages/dar_collection_review/DarCollectionReview.jsx

```diff
@@ -75,14 +77,58 @@ const tabsForUser = (user, buckets, adminPage = false) => {
   if (!isEmpty(myChairVotes)) {
     updatedTabs.chairVote = 'Chair Vote'
   }
+  if (userIsDacUser(user)) {
+    updatedTabs.votingHistory = 'Voting History'
+  }
   return updatedTabs
 }
 
+const getApprovedDatasetsFromLatestDar = (darCollection, dacIds) => {
+  // most recently submitted DAR
+  const dars = Object.values(darCollection.dars)
+  if (dars.length === 0) return []
+  const mostRecentDar = dars.filter(d => !d.draft).reduce((latest, current) =>
+    new Date(current.submissionDate) > new Date(latest.submissionDate) ? current : latest,
+  )
+
+  // most recent closed election
+  const elections = Object.values(mostRecentDar.elections || {})
+  const closedDataAccessElections = elections.filter(e => e.electionType == 'DataAccess' && e.status === 'Closed')
+  if (closedDataAccessElections.length === 0) return []
+  const latestElection = closedDataAccessElections.reduce((latest, current) =>
+    new Date(current.createDate) > new Date(latest.createDate) ? current : latest,
+  )
+
+  // datasets approved in that election
+  const approvedDatasetIds = []
+  Object.values(latestElection.votes || {}).forEach((vote) => {
+    if ((vote.type === 'FINAL' || vote.type === 'RADAR_APPROVE') && vote.vote === true) {
+      approvedDatasetIds.push(latestElection.datasetId)
+    }
+  })
+
+  // filter for this DAC if this is a DAC page, and get dataset names
+  const approvedDatasetNames = [...new Set(approvedDatasetIds)].filter((datasetId) => {
+    if (dacIds.length === 0) return true // admin page
+    const dataset = darCollection.datasets?.find(ds => ds.datasetId === datasetId)
+    return dacIds.includes(dataset?.dacId)
+  }).map((datasetId) => {
+    const dataset = darCollection.datasets?.find(ds => ds.datasetId === datasetId)
+    return dataset?.name
+  })
+  return approvedDatasetNames || []
+}
+
+const userIsDacUser = (user) => {
+  return user.roles?.some(role => role.roleId === 2 || role.roleId === 1)
```

Comment:
I'm wondering if you should just let the backend do the filtering and display what's returned.  There are a number of edge cases because the user can be in multiple DACs and have different roles in each DAC.  Given you have a ticket to cover the issue you found, I'm not sure it makes sense to try to filter like this in the UI.